arviz_dashboard/__init__.py

```diff
@@ -1,4 +1,5 @@
 """Bayesian Dashboards built on top of Panel"""
 __version__ = "0.0.1a"
 
-from .elpd import dashboard_elpd
\ No newline at end of file
+from .elpd import dashboard_elpd
+from arviz_dashboard.marginal.one_d import posterior_marginal1d_dashboard
```

Comment:
Not necessary to change now, but we should think of shorter names for the dashboard functions and if we are going to import arviz_dashboard as `azd` or something similar then we could omit the `dashboard` from the names.

Reply:
I like this suggestion, and will make the change before mergining.

I'll also take all the suggestions you mentioned above and turn them into tickets for enhancements.